config/rooms.cfg

```diff
@@ -35,6 +35,8 @@
 # LibraryCooldownWorkMin            (uint)   How many turns a creature will stay idle after working minimum
 # LibraryCooldownWorkMax            (uint)   How many turns a creature will stay idle after working maximum
 # LibrarySkillPointsBook            (uint)   How many points have to be researched to create a book
+# PortalCooldownSpawnMin            (uint)   How many turns a portal will not spawn any creature after spawning minimum
```

Comment:
Yep, you understood correctly. I guess I used copy/paste too much and in this case, it is not very meaningfull ^^